### DISCUSSION TITLE
Does removing Flameshot delete images uploaded to Imgur?

### DISCUSSION THREAD

**Discussion by ghost** [flameshot-org/flameshot#3157] [Q&A]
Flameshot automatically uploads screenshots to Imgur. If I remove Flameshot from the PC, will the images uploaded to Imgur be automatically deleted or will my images still be stored on Imgur?

**Comment by ghost** [comment]
https://github.com/flameshot-org/flameshot/issues/3156#issuecomment-1503585442

**Reply by mmahmoudian** [reply]
Btw, Flameshot does not automatically upload images. You have to click on the upload button and then you should again approve the upload in the confirmation dialog.
